# Fix paths
--- /usr/share/llvm-3.8/cmake/LLVMConfig.cmake  2020-11-23 10:06:50.536692742 +0000
+++ /usr/share/llvm-3.8/cmake/LLVMConfig.cmake  2020-11-23 10:10:22.746700044 +0000
@@ -168,11 +168,11 @@

 set(LLVM_LIBDIR_SUFFIX )

-set(LLVM_INCLUDE_DIRS "${LLVM_INSTALL_PREFIX}/include")
-set(LLVM_LIBRARY_DIRS "${LLVM_INSTALL_PREFIX}/lib${LLVM_LIBDIR_SUFFIX}")
+set(LLVM_INCLUDE_DIRS "${LLVM_INSTALL_PREFIX}/include/llvm-3.8")
+set(LLVM_LIBRARY_DIRS "${LLVM_INSTALL_PREFIX}/lib${LLVM_LIBDIR_SUFFIX}/llvm-3.8/lib")
 set(LLVM_DEFINITIONS "-D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS")
-set(LLVM_CMAKE_DIR "${LLVM_INSTALL_PREFIX}/share/llvm/cmake")
-set(LLVM_TOOLS_BINARY_DIR "${LLVM_INSTALL_PREFIX}/bin")
+set(LLVM_CMAKE_DIR "${LLVM_INSTALL_PREFIX}/share/llvm-3.8/cmake")
+set(LLVM_TOOLS_BINARY_DIR "${LLVM_INSTALL_PREFIX}/lib${LLVM_LIBDIR_SUFFIX}/llvm-3.8/bin")

 if(NOT TARGET LLVMSupport)
   include("${LLVM_CMAKE_DIR}/LLVMExports.cmake")

# Fix import prefix and remove polly as it's not a part of the LLVM package
--- /usr/share/llvm-3.8/cmake/LLVMExports-relwithdebinfo.cmake  2020-11-23 10:10:46.280034183 +0000
+++ /usr/share/llvm-3.8/cmake/LLVMExports-relwithdebinfo.cmake  2020-11-23 10:13:04.810038954 +0000
@@ -5,6 +5,9 @@
 # Commands may need to know the format version.
 set(CMAKE_IMPORT_FILE_VERSION 1)

+# Fix import prefix
+set(_IMPORT_PREFIX "/usr/lib${LLVM_LIB_PREFIX}/llvm-3.8/")
+
 # Import target "LLVMSupport" for configuration "RelWithDebInfo"
 set_property(TARGET LLVMSupport APPEND PROPERTY IMPORTED_CONFIGURATIONS RELWITHDEBINFO)
 set_target_properties(LLVMSupport PROPERTIES
@@ -1164,36 +1167,6 @@
 list(APPEND _IMPORT_CHECK_TARGETS LLVMLibDriver )
 list(APPEND _IMPORT_CHECK_FILES_FOR_LLVMLibDriver "${_IMPORT_PREFIX}/lib/libLLVMLibDriver.a" )

-# Import target "PollyISL" for configuration "RelWithDebInfo"
-set_property(TARGET PollyISL APPEND PROPERTY IMPORTED_CONFIGURATIONS RELWITHDEBINFO)
-set_target_properties(PollyISL PROPERTIES
-  IMPORTED_LINK_INTERFACE_LANGUAGES_RELWITHDEBINFO "C"
-  IMPORTED_LOCATION_RELWITHDEBINFO "${_IMPORT_PREFIX}/lib/libPollyISL.a"
-  )
-
-list(APPEND _IMPORT_CHECK_TARGETS PollyISL )
-list(APPEND _IMPORT_CHECK_FILES_FOR_PollyISL "${_IMPORT_PREFIX}/lib/libPollyISL.a" )
-
-# Import target "Polly" for configuration "RelWithDebInfo"
-set_property(TARGET Polly APPEND PROPERTY IMPORTED_CONFIGURATIONS RELWITHDEBINFO)
-set_target_properties(Polly PROPERTIES
-  IMPORTED_LINK_INTERFACE_LANGUAGES_RELWITHDEBINFO "CXX"
-  IMPORTED_LOCATION_RELWITHDEBINFO "${_IMPORT_PREFIX}/lib/libPolly.a"
-  )
-
-list(APPEND _IMPORT_CHECK_TARGETS Polly )
-list(APPEND _IMPORT_CHECK_FILES_FOR_Polly "${_IMPORT_PREFIX}/lib/libPolly.a" )
-
-# Import target "LLVMPolly" for configuration "RelWithDebInfo"
-set_property(TARGET LLVMPolly APPEND PROPERTY IMPORTED_CONFIGURATIONS RELWITHDEBINFO)
-set_target_properties(LLVMPolly PROPERTIES
-  IMPORTED_LOCATION_RELWITHDEBINFO "${_IMPORT_PREFIX}/lib/LLVMPolly.so"
-  IMPORTED_NO_SONAME_RELWITHDEBINFO "TRUE"
-  )
-
-list(APPEND _IMPORT_CHECK_TARGETS LLVMPolly )
-list(APPEND _IMPORT_CHECK_FILES_FOR_LLVMPolly "${_IMPORT_PREFIX}/lib/LLVMPolly.so" )
-
 # Import target "LTO" for configuration "RelWithDebInfo"
 set_property(TARGET LTO APPEND PROPERTY IMPORTED_CONFIGURATIONS RELWITHDEBINFO)
 set_target_properties(LTO PROPERTIES
